src/components/button/WindowCloseButton.tsx

```diff
@@ -0,0 +1,20 @@
+import { IconButton, IconButtonProps } from "@mui/material";
+import React, { ReactElement } from "react";
+import CloseIcon from "@mui/icons-material/Close";
+
+function WindowCloseButton({ onClick, ...otherProps }: IconButtonProps): ReactElement {
+  return (
+    <IconButton
```

Comment:
Tuossa exportissa käytät React.forwarRefiä. Jos sitä käyttää niin kannattaa myös viedä ref "maaliin" saakka. Ehdotuksessani sen pitäisi olla kunnossa. Vaihtoehtoisesti voit katsoa mallia miten Button-komponentissa on refin kanssa toimittu.
```suggestion
function WindowCloseButton(
  { onClick, ...otherProps }: IconButtonProps,
  ref: React.ForwardedRef<HTMLButtonElement>
): ReactElement {
  return (
    <IconButton
      ref={ref}
```